Sources_v3/StreamChatUI/ChatChannelList/ChatUnreadCountBubbleView.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class UnreadCountBubbleView: UIView {
+    
+    let inset: CGFloat = 4
+    
+    open override var intrinsicContentSize: CGSize {
+        .init(width: max(unreadCountLabel.intrinsicContentSize.width + inset*2, 16), height: 16)
+    }
```

Comment:
Maybe we can mange this by having a `width constraint` on `UnreadCountBubbleView` that will be **>= 16**? It might eliminate the need of invoking `invalidateIntrinsicContentSize` (not checked)